src/modules/rlang.rs

```diff
@@ -0,0 +1,96 @@
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::rlang::RLangConfig;
+use crate::formatter::StringFormatter;
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("rlang");
+    let config: RLangConfig = RLangConfig::try_load(module.config);
+
+    let is_r_project = context
+        .try_begin_scan()?
+        .set_files(&config.detect_files)
+        .set_extensions(&config.detect_extensions)
+        .set_folders(&config.detect_folders)
+        .is_match();
+    if !is_r_project {
+        return None;
+    }
+
+    let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+        formatter
+            .map_meta(|var, _| match var {
+                "symbol" => Some(config.symbol),
+                _ => None,
+            })
+            .map_style(|variable| match variable {
+                "style" => Some(Ok(config.style)),
+                _ => None,
+            })
+            .map(|variable| match variable {
+                "version" => get_r_version(context).map(Ok),
```

Comment:
Can you add version formatting to this module? #2611

Reply:
Just added it!